images/sdk/configs/latest.melange.yaml

```diff
@@ -58,13 +58,9 @@ pipeline:
       git clone https://github.com/chainguard-dev/apko.git
       (cd apko && make apko install)
 
+      # move all Go-built stuff into /usr/bin, remove uneeded files
+      mv "${GOPATH}/bin/*" "${DESTDIR}/usr/bin"
```

Comment:
I don't think we need this part

Reply:
just pushed up a new version. trying to get goimports into the package

Reply:
Why?  It's not a hard requirement for building, and I don't think people really use this image to build apko and melange itself, but rather to make use of them.

Reply:
> I don't think people really use this image to build apko and melange itself

i do personally :)

Reply:
Carry on, then :)